helm/prometheus-rules/templates/kaas/rocket/alerting-rules/blackbox-exporter.cloud-provider-api.rules.yml

```diff
@@ -0,0 +1,26 @@
+# This rule is applied to all management clusters but it is only active if blackbox
+# exporter is deployed and configured with a scrape job named 'http-cloud-provider-api'
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: rocket-onprem-cloud-provider-api
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: rocket-onprem-cloud-provider-api
+    rules:
+    - alert: OnPremCloudProviderAPIIsDown
+      annotations:
+        description: '{{` blackbox-exporter on {{ $labels.cluster_id}} is unable to connect to the on-prem cloud provider API.`}}'
+        runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/onprem-cloud-provider-api-is-down/
+      expr: probe_success{cluster_type="management_cluster",job="prometheus-blackbox-exporter",target="http-cloud-provider-api"} == 0
+      for: 5m
+      labels:
+        area: kaas
+        severity: page
+        team: rocket
+        topic: network
+        all_pipelines: "true"
```

Comment:
I added this so that it also pages for testing installations - in the past we've only discovered that testing installations are broken when the tests start failing